### PR TITLE
Add documentation for flutter usage

### PR DESCRIPTION
**Pull request by tomasatisocco**
Add flutter documentation, it has some little diferences with the kotlin documentation that can help flutter developers that needs to use this awesome package in the native layer more easily

### CODE REVIEW

**Review by paulschwarz — COMMENTED** [review]
A few typos. The quality of English needs to be improved.

**Comment by tomasatisocco** [comment]
> A few typos. The quality of English needs to be improved.

Fixed, if you find some more typos that needs to be improved please let me know.
I'm not native english speaker sorry for the mistakes.

**Comment by cdimascio** [comment]
In the Android case, /assets is not a relative path. Please revert that change

File: README.md [hunk 1]
```diff
@@ -85,24 +85,39 @@ dotenv["MY_ENV_VAR1"]
...
-**Note:** The above configuration is required because dot files in `/assets` do not appear to resolve on Android. *(Seeking recommendations from the Android community on how `dotenv-kotlin` configuration should work in order to provide the best experience for Android developers)*
+**Note:** The above configuration is required because dot files in `./assets` do not appear to resolve on Android. *(Seeking recommendations from the Android community on how `dotenv-kotlin` configuration should work in order to provide the best experience for Android developers)*
 
 Alternatively, if you are using Provider `android.resource` you may specify
 
 ```
  directory = "android.resource://com.example.dimascio.myapp/raw"
 ```
 
+## Fluter Usage
```
Comment (cdimascio) [inline README.md]: typo `Flutter` not `Fluter`

Comment (tomasatisocco) [reply README.md]: Solved and reverted the unnecessary changes. Sorry for the delay! Have been very busy months at the work.

**Pull request merged**